Sources/ComposableArchitecture/Documentation.docc/Tutorials/MeetTheComposableArchitecture/01-Essentials/01-YourFirstFeature/01-01-YourFirstFeature.tutorial

```diff
@@ -24,6 +24,11 @@
       @Step {
         Create a new Swift file named CounterFeature.swift and import the Composable Architecture.
         @Code(name: "CounterFeature.swift", file: 01-01-01-code-0001.swift)
+        
+        > Note: Before importing the library you will need to depend on it by adding it to your 
```

Comment:
Maybe link to here? https://github.com/pointfreeco/swift-composable-architecture#installation

Reply:
Good suggestion, done!